detect/detect.go

```diff
@@ -403,15 +415,33 @@ func (d *Detector) DetectFiles(source string) ([]report.Finding, error) {
 					return nil
 				}
 				if fInfo.Mode().IsRegular() {
-					paths <- path
+					paths <- scanTarget{
+						Path:    path,
+						Symlink: "",
+					}
+				}
+				if fInfo.Mode().Type() == fs.ModeSymlink && d.FollowSymlinks {
+					realPath, err := filepath.EvalSymlinks(path)
+					if err != nil {
+						return err
+					}
+					realPathFileInfo, _ := os.Stat(realPath)
+					if realPathFileInfo.IsDir() {
+						fmt.Printf("Found symlinked directory: %s -> %s [skipping]\n", path, realPath)
```

Comment:
@RickyGrassmuck could we change this to a debug log instead? 
```suggestion
						log.Debug().Msgf("found symlinked directory: %s -> %s [skipping]", path, realPath)
```

Reply:
Ahh, missed one, all fixed up!